templates.go

```diff
@@ -191,7 +193,6 @@ func getTemplates() *template.Template {
 <script>
 function logout() {
         var url = "https://api.getcoco.buzz/v1/auth/logout";
-
```

Comment:
- remove all the consoles in the below template

Reply:
handled